app/views/unsubmitted_course_alert_mailer/email.html.haml

```diff
@@ -22,4 +22,4 @@
                 Best,
                 %br
                 %em
-                  =@classroom_program_manager.username
+                  =@classroom_program_manager&.username || 'The system'
```

Comment:
I guess "Classroom Program Manager" is the best option here. That would be a usable fallback if for some reason we removed the special user in production.

Reply:
Done :)